field/arena_notifiers.go

```
@@ -379,6 +381,25 @@ func (arena *Arena) generateScoringStatusMessage() any {
 	}
 }
 
+func (arena *Arena) generateLedModeMessage() interface{} {
+    redMode, blueMode := arena.Leds.GetModes()
+
+	log.Printf("Generating LED mode message with RedMode=%d and BlueMode=%d", redMode, blueMode)
+
+    // Only notify if mode has actually changed
+    if redMode == arena.lastRedLedMode && blueMode == arena.lastBlueLedMode {
```

golangci-lint (2.12.2)

[error] SA9003: empty branch

(staticcheck)